shared/templates/template_OVAL_package_installed

```diff
@@ -4,15 +4,25 @@
     <metadata>
       <title>Package %PKGNAME% Installed</title>
       <affected family="unix">
-        <platform>Debian 8</platform>
+        <platform>multi_platform_all</platform>
       </affected>
-      <description>The DEB package %PKGNAME% should be installed.</description>
+      <description>The {{{ pkg_system|upper }}} package %PKGNAME% should be installed.</description>
```

Comment:
But then here it be expanded to `DPKG package`, not `DEB package`.

Reply:
It may actually be `DPKG package`, but I think I never saw like this.

Reply:
is that a problem? if anything DPKG package is more correct because DEB is short for debian so it doesn't apply on Ubuntu.